config/armada/config.yaml

```diff
@@ -98,17 +98,10 @@ submission:
   defaultActiveDeadline: "72h" # 3 days.
   defaultActiveDeadlineByResourceRequest:
     nvidia.com/gpu: "336h" # 14 days.
-eventRetention:
```

Comment:
All config changes in this PR are to remove yaml that doesn't match any config defined in the corresponding struct.